lib/browser/mqtt.ts

```diff
@@ -310,7 +311,7 @@ export class MqttClientConnection extends BufferedEventEmitter {
 
         const will = this.config.will ? {
             topic: this.config.will.topic,
-            payload: normalize_payload(this.config.will.payload),
+            payload: normalize_payload_to_buffer(this.config.will.payload),
```

Comment:
This seems to be a breaking change where `normalize_payload_to_buffer` always return a buffer, while `normalize_payload` could return any type. Should we use `normalize_payload_to_buffer` for all payload in that case?

Reply:
Never mind, my fault. The will would only be accessed internally, it should not affect any user.